app/server.js

```diff
@@ -0,0 +1,67 @@
+const cluster = require('express-cluster');
+const express = require('express');
+const bodyParser = require('body-parser');
+const errorhandler = require('errorhandler');
```

Comment:
error handler haven't to be installed on production. require it only on production environment